components/scream/cime_config/namelist_defaults_scream.xml

```diff
@@ -74,29 +74,41 @@ tweaking their $case/namelist_scream.xml file.
            set of selectors. A parameter element with no selectors, which means it controls the default param value, must
            come before parameter elements with selectors. If multiple parameter elements
            are matched, then the match the occurs last takes precendence.
+
         2) The first matched parameter element encountered for a parameter will define the default metadata
            for that parameter. All subsequent parameter elements will assume the same
            metadata as the first unless an explicit override occurs.
            User changes to the case XML file must be compatible with the metadata.
+
         3) For a parameter whose elements all have selectors, if none of the selectors
            are matched, then the parameter will be omitted from the case XML
            file ($case/namelist_scream.xml).
+
         4) Parameter types will be inferred. You can override the inferred type
            via the 'type' metadata attribute. Types are:
              logical: will be inferred from any case of true/false
              integer: will be inferred from any sequence of 0-9
              real   : will be inferred from any numeric expression with a decimal point or scientific notation
+             file   : A string that denotes a file path that CIME should download
+                      NOTE: All file parameters will be symlinked to ${RUNDIR}/data from the download area.
              string : everything else
+
+             array(one of the above): An array of one of the above types
```

Comment:
We might say that both `[a,b,c]` and `(a,b,c)` syntaxes are supported. At least I think they are...